app/Http/Controllers/ScreenshotController.php

```diff
@@ -12,30 +12,30 @@
 
 class ScreenshotController extends Controller
 {
-    public function show($gameid, $screenid){
+    public function show($gameid, $screenid) {
         $s = Screenshot::whereGameId($gameid)->where('screenshot_id', $screenid)
             ->first();
 
         $storagePath = '';
 
         //Prüfen ob Screenshots vorhanden sind
-        if(is_null($s)){//Es sind keine Screenshots vorhanden
+        if (is_null($s)) {//Es sind keine Screenshots vorhanden
             $storagePath = public_path().'/assets/no_image.png';
-        }else{//Es sind Screenshots vorhanden
+        }else {//Es sind Screenshots vorhanden
             $storagePath = \Storage::get($s->filename);
         }
 
         return Image::make($storagePath)->response();
     }
 
-    public function create($gameid, $screenid){
+    public function create($gameid, $screenid) {
         return view('screenshots.create', [
             'gameid' => $gameid,
             'screenid' => $screenid,
         ]);
     }
 
-    public function upload(Request $request, $gameid, $screenid){
+    public function upload(Request $request, $gameid, $screenid) {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->